source/server/http/admin.cc

```diff
@@ -761,11 +761,10 @@ Http::Code AdminImpl::handlerCerts(absl::string_view, Http::HeaderMap& response_
   envoy::admin::v2alpha::Certificates certificates;
   server_.sslContextManager().iterateContexts([&](const Ssl::Context& context) -> void {
     envoy::admin::v2alpha::Certificate& certificate = *certificates.add_certificates();
-
     envoy::admin::v2alpha::CertificateDetails* ca_certificate = certificate.mutable_ca_cert();
-    MessageUtil::loadFromJson(context.getCaCertInformation(), *ca_certificate);
+    ca_certificate->MergeFrom(*context.getCaCertInformation());
```

Comment:
I think you can just assign, instead of doing MergeFrom, given that ca_certificate should be empty (eg we can replace instead of merge).